src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -304,15 +304,9 @@ def list_vault(client, resource_group_name=None):
     return list(vault_list)
 
 
-# pylint: disable=inconsistent-return-statements
 def _get_current_user_object_id(graph_client):
-    from azure.cli.command_modules.role import GraphError
-    try:
-        current_user = graph_client.signed_in_user_get()
-        if current_user and current_user.get('id', None):  # pylint:disable=no-member
-            return current_user['id']  # pylint:disable=no-member
-    except GraphError:
-        pass
```

Comment:
We got tripped by some code written almost 6 years ago (cb0faa82bc5db56a15d0c03e4dad5cf040ae9d83)!